base/server/src/main/java/com/netscape/cmscore/base/ArgBlock.java

```diff
@@ -271,11 +254,10 @@ public BigInteger getValueAsBigInteger(String n, BigInteger def) {
      */
     @Override
     public Object getValue(Object n) throws EBaseException {
-        if (mArgs.get(n) != null) {
-            return mArgs.get(n);
-        } else {
+        if (mArgs.get(n) == null) {
             throw new EBaseException(CMS.getUserMessage("CMS_BASE_ATTRIBUTE_NOT_FOUND", (String) n));
         }
+        return mArgs.get(n);
```

Comment:
Not a big issue here, but in general we want to avoid doing the lookup twice:
```
Object result = mArgs.get(n);
if (result == null) {
    throw ....
}
return result;
```